runner/check.go

```diff
@@ -38,13 +38,15 @@ func (c *Check) Run(ctx context.Context, role string) ([]byte, int, error) {
 	timeout, err := time.ParseDuration(c.Timeout)
 	if err != nil {
 		logrus.Errorf("error reading timeout %s. Using default timeout 5sec", err)
-		timeout = time.Second*5
+		timeout = time.Second * 5
 	}
 
 	newCtx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
-	stdout, stderr, code, err := exec.Output(newCtx, c.Cmd...)
+	cmd := exec.CommandContext(newCtx, c.Cmd...)
+	cmd.Env = []string{"PATH=/bin:/usr/bin:/opt/mesosphere/bin"} // TODO(branden)
```

Comment:
maybe we should extend the original environment variables instead of overriding the whole thing?
i think this will remove the original and replace with only one variable PATH

Also, maybe we should make it a constant?